src/app/main/service/jwt/jwt.service.ts

```
@@ -38,7 +38,8 @@ export class JwtService {
     if (accessToken != null) {
       const payload = accessToken.split('.')[1];
       const decodedPayload = window.atob(payload);
-      return JSON.parse(decodedPayload).role[0];
+      const payload = JSON.parse(decodedPayload);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'payload'. Consider to delete it or rename it.

'payload' is defined here:


(lint/suspicious/noRedeclare)